src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/help.py

```diff
@@ -15,7 +15,39 @@
             type: group
             short-summary: Commands to manage servers.
             """
+helps['sql server create'] = """
+            type: command
+            short-summary: Create a new Azure SQL server
+            """
 helps['sql server list'] = """
             type: command
-            short-summary: List the SQL Server belongs to given resource grounp
+            short-summary: List the SQL Server belongs to given resource group.
+            """
+helps['sql server update'] = """
+            type: command
+            short-summary: Update a Azure SQL server.
+            """
+helps['sql server firewall'] = """
+            type: group
+            short-summary: Commands to manage Azure SQL server's firewall rules
+            """
+helps['sql server firewall create'] = """
+            type: command
+            short-summary: Create an Azure SQL server firewall rule
+            """
+helps['sql server firewall update'] = """
+            type: command
+            short-summary: Update an Azure SQL server firewall rule
+            """
+helps['sql server firewall show'] = """
+            type: command
+            short-summary: Show the details of an Azure SQL server firewall rule
+            """
+helps['sql server firewall list'] = """
+            type: command
+            short-summary: List the Azure SQL server firewall rules
+            """
+helps['sql server service-objective'] = """
+            type: group
+            short-summary: Commands to manage Azure SQL server's service objectives
```

Comment:
Since these commands are really just cataloging (list and get details) the generic group help probably isn't best here. There's literally nothing you can manage about service objectives.